metrics/cache/cache_test.go

```diff
@@ -0,0 +1,252 @@
+package cache_test
+
+import (
+	"strings"
+
+	dto "github.com/prometheus/client_model/go"
+)
+
+//
```

Comment:
uncomment

Reply:
Done in https://github.com/google/cadvisor/pull/2974/commits/a587512437b8dda97203ffee4757a2b9ea563c44.